corehq/util/tests/test_celery_utils.py

```diff
@@ -0,0 +1,112 @@
+from datetime import datetime, time
+
+from celery.concurrency.base import BasePool
+from django.test import SimpleTestCase, override_settings
+import mock
+
+from corehq.util.celery_utils import LoadBasedAutoscaler, OffPeakLoadBasedAutoscaler
+
+
+class Object(object):
+    pass
+
+
+class MockPool(BasePool):
```

Comment:
This setup is pulled from celery tests https://github.com/celery/celery/blob/v3.1.25/celery/tests/worker/test_autoscale.py